docs/conf.py

```diff
@@ -221,8 +221,8 @@
 # https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html
 #
 intersphinx_mapping = {
-    "plonedocs": ("https://docs.plone.org/", None),
-    "plone6docs": ("https://6.dev-docs.plone.org/", None),
+    "plonedocs": ("https://5.docs.plone.org/", None),
```

Comment:
I'd like to use an explicit version number going forward. This change will require a global find and replace. [My search turned up only this one instance](https://github.com/plone/training/search?q=plonedocs), so it should be trivial.

```suggestion
    "plone5docs": ("https://5.docs.plone.org/", None),
```